### ISSUE TITLE
Missing NPC Dire Maul

### ISSUE THREAD

**Issue by Nostealth** [dalaranwow/dalaran-wow#4944]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
There is supposed to be an NPC in Dire Maul, Knot Thimblejack.I was farming reputation with Steamweedle Cartel and after a few deployed fixes the NPC disappeared.
**Expected behaviour**: Tell us what should happen instead
He's always supposed to be in the same spot, unless you free him, which is one of the quests you can make inside Dire Maul.
**Steps to reproduce the problem**:

1. I went to Dire Maul to make reputation with the Steamweedle Cartel.
2. I got to the spot where the NPC was supposed to be. I tried clearing the instance and reseting it.
3. Nothing changed.The NPC is nowhere to be found.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Nostealth.
I can confirm the missing creature on PTR, but he is not alone there are more than 1 missing there.

From PTR:
![image](https://user-images.githubusercontent.com/6797611/31944107-a3ce4208-b8d3-11e7-82dc-40c9ffa8c682.png)

From TC:
![image](https://user-images.githubusercontent.com/6797611/31944120-af223902-b8d3-11e7-90cc-370b4c0c212a.png)

**Comment by TeridanDW** [comment]
Spawning the creature, guid 56804 does not exits on PTR, so it must have been deleted by mistake with few more creatures montioned above.

**Suggested fix:**
``` sql
DELETE FROM `creature` WHERE `guid`=56804;
INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
(56804,14338,429,0,0,1,1,0,0,591.98,598.8,-4.57,3.94,644,0,0,2215,0,0,0,0,0,'',0);
```

**Comment by TeridanDW** [comment]
Works as intended on PTR.

**Comment by Chenocide** [comment]
The NPC didn't drop the chest when I just killed him. It contains several recipes, so that's a bummer.
